include/my_dbug.h

```diff
@@ -127,6 +127,7 @@ extern  const char* _db_get_func_(void);
 #define DBUG_END()  _db_end_ ()
 #define DBUG_LOCK_FILE _db_lock_file_()
 #define DBUG_UNLOCK_FILE _db_unlock_file_()
+#define DBUG_ASSERT(A) assert(A)
```

Comment:
@ldonoso I believe Oracle deliberately got rid of `DBUG_OFF` and `DBUG_ASSERT`, so they should not be restored here.
c6d0812b430 Bug #32354435 BACKPORT TO 5.7: BUG #32216281 REMOVE DBUG_OFF AND DBUG_ASSERT
14054f54ead Bug #32354435 BACKPORT TO 5.7: BUG #32216281 REMOVE DBUG_OFF AND DBUG_ASSERT
6889e5c00a5 Bug #32354435 BACKPORT TO 5.7: BUG #32216281 REMOVE DBUG_OFF AND DBUG_ASSERT

Reply:
Already found that it was removed in 5f0d84e